lib/taste_tester/host.rb

```diff
@@ -23,12 +25,16 @@
 require 'taste_tester/noop'
 require 'taste_tester/locallink'
 require 'taste_tester/tunnel'
+require 'taste_tester/exceptions'
 
 module TasteTester
   # Manage state of the remote node
   class Host
     include TasteTester::Logging
 
+    TASTE_TESTER_CONFIG = 'client-taste-tester.rb'
+    USER_PREAMBLE = '# TasteTester by '
```

Comment:
This is exactly the same string as used today: https://github.com/facebook/taste-tester/blob/master/lib/taste_tester/host.rb#L227

I agree this is not grammatically sound. Changing it would involve a backward compatibility hack else we'd lose the mutual exclusion aspect entirely. I opt to be laz^Wefficient here and focus on the main race condition problems.

Reply:
Yup, agreed. Just mis-remembered.